packages/adapter-vercel/files/serverless.js

```diff
@@ -23,7 +23,7 @@ export default async (req, res) => {
 		request = await getRequest({ base: `https://${req.headers.host}`, request: req });
 	} catch (err) {
 		res.statusCode = err.status || 400;
-		return res.end(err.reason || 'Invalid request body');
+		return res.end(err.reason || err.toString() || 'Invalid request body');
```

Comment:
In the other places this was changed to `err.reason`, why not here? Can we safely assume that `toString` as a method is always present on the error object?

In general (again, not well-versed, just asking questions): Which errors end up in here, which would have `reason` set? Is this just an oversight from previous iterations?

Reply:
I myself don't understand where this `reason` can come from, but I don't know anything about this adapter, so I decided to leave everything as it is. In my opinion, all objects have toString() except Object.create(null).  But nevertheless it is possible to do so `err.toString?.()`

 I think it's just an oversight of previous iterations.

Reply:
Done